composer.json

```diff
@@ -14,47 +14,46 @@
     "minimum-stability": "stable",
     "require": {
         "php": "~5.6|~7.0",
-        "symfony/symfony": "3.4.*",
-        "symfony/assetic-bundle": "~2.3",
-        "symfony/swiftmailer-bundle": "~2.3",
-        "symfony/monolog-bundle": "~2.4",
-        "twig/twig": "^1.35",
-        "twig/extensions": "~1.5",
-        "doctrine/doctrine-migrations-bundle": "^1.1",
-        "doctrine/doctrine-bundle": "^1.6",
-        "doctrine/dbal": "~2.5",
-        "doctrine/orm": "~2.5",
-        "doctrine/common": "~2.5",
+        "beberlei/assert": "^2.0",
         "doctrine/cache": "~1.4",
+        "doctrine/common": "~2.5",
+        "doctrine/dbal": "~2.5",
+        "doctrine/doctrine-bundle": "^1.6",
+        "doctrine/doctrine-migrations-bundle": "^1.1",
         "doctrine/migrations": "~1.4",
-        "sensio/distribution-bundle": "^5.0",
-        "sensio/framework-extra-bundle": "~3.0",
-        "incenteev/composer-parameter-handler": "~2.0",
-        "nelmio/security-bundle": "~1.4",
-        "mopa/bootstrap-bundle": "^3.2",
-        "twbs/bootstrap": "^3.3.0",
+        "doctrine/orm": "~2.5",
         "fortawesome/font-awesome": "~4.2.0",
+        "graylog2/gelf-php": "^1.5",
+        "incenteev/composer-parameter-handler": "~2.0",
         "jms/translation-bundle": "~1.3.0",
+        "mopa/bootstrap-bundle": "^3.2",
         "mopa/composer-bridge": "~1.5",
+        "nelmio/security-bundle": "~1.4",
+        "openconext/monitor-bundle": "^1.0",
+        "sensio/distribution-bundle": "^5.0",
+        "sensio/framework-extra-bundle": "~3.0",
         "surfnet/messagebird-api-client-bundle": "^4.0",
         "surfnet/stepup-bundle": "^4.0",
         "surfnet/stepup-saml-bundle": "^4.1",
         "surfnet/stepup-u2f-bundle": "dev-develop",
         "surfnet/yubikey-api-client-bundle": "^3.0",
-        "openconext/monitor-bundle": "^1.0",
-        "beberlei/assert": "^2.0",
-        "webmozart/assert": "^1.2",
-        "graylog2/gelf-php": "^1.5"
+        "symfony/assetic-bundle": "~2.3",
+        "symfony/monolog-bundle": "^3.0",
+        "symfony/swiftmailer-bundle": "~2.3",
+        "symfony/symfony": "3.4.*",
+        "twbs/bootstrap": "^3.3.0",
+        "twig/extensions": "~1.5",
+        "twig/twig": "1.41.0",
```

Comment:
Why is this hardcoded?

Reply:
See: https://github.com/OpenConext/Stepup-Gateway/pull/189/commits/c585d17b9a50f6168127161279acad96eed914eb